pyproject.toml

```diff
@@ -7,7 +7,7 @@ license = "Apache-2.0"
 
 [tool.poetry.dependencies]
 python = "^3.10"
-teslajsonpy = "^3.7.4"
+teslajsonpy = "3.7.5"
```

Comment:
Thanks, actually you can leave the carat in front; it doesn't matter in the end.

Reply:
Done!